### ISSUE TITLE
udemy get to know youve downloaded the course

### ISSUE THREAD

**Issue by chickenleaf** [FaisalUmair/udemy-downloader-gui#554]
udemy gets to know youve downloaded the course and so it takes action on our account

**Comment by Animizio** [comment]
Any ideas to hide that informations to udemy?

**Comment by UltramarineLights** [comment]
Yes unfortunately I also noticed the same for a downloaded course. Any updates on this?

**Comment by Detective-Khalifah** [comment]
Terrible!

**Comment by Detective-Khalifah** [comment]
> udemy gets to know youve downloaded the course and so it takes action on our account

Is the action aggressive?

**Comment by Detective-Khalifah** [comment]
> Yes unfortunately I also noticed the same for a downloaded course. Any updates on this?

What happened?

**Comment by codev911** [comment]
i think add a user agent manipulation script will better, if you manipulate user agent maybe work

**Comment by ghana7989** [comment]
> i think add a user agent manipulation script will better, if you manipulate user agent maybe work

Can you be more specific and tell the process?

**Comment by MRN109** [comment]
Just use a Vpn i have bought and downloaded courses for a long time now.
